lib/src/cli.rs

```diff
@@ -971,6 +989,17 @@ async fn run_from_opt(opt: Opt) -> Result<()> {
                 crate::install::install_to_existing_root(opts).await
             }
             InstallOpts::PrintConfiguration => crate::install::print_configuration(),
+            InstallOpts::EnsureCompletion { sysroot, stateroot } => {
+                let rootfs = &Dir::open_ambient_dir("/", cap_std::ambient_authority())?;
+                if let Some(sysroot) = sysroot {
+                    let stateroot = stateroot.as_deref().ok_or_else(|| {
+                        anyhow::anyhow!("Expected stateroot when --sysroot is set")
+                    })?;
```

Comment:
```suggestion
                    let stateroot = stateroot.as_deref().unwrap();
```

Reply:
This isn't true with just `requires`...the only way to make this work would be to split up the argument. Which I'll do instead.